CHANGELOG.rst

```diff
@@ -0,0 +1,35 @@
+#########
+Changelog
+#########
+All notable changes to the kytos ui project will be documented in this file.
+
+UNRELEASED - Under development
+******************************
+Added
+=====
+
+Changed
+=======
+
+Deprecated
+==========
+
+Removed
+=======
+
+Fixed
+=====
+
+Security
+========
+
+Changed
+=======
+
+
+[2020.2b2] - "itamar" beta2 - 2020-10-23
+****************************************
```

Comment:
Currently, the versioning for the UI is like the versioning for the NApps: a separate MAJOR.MINOR numbering scheme that does not follow the 3 main projects YEAR.SEMESTER versioning.


```suggestion
[1.3.2] - 2020-10-23
********************
```